contrib/epee/include/net/abstract_tcp_server2.inl

```diff
@@ -985,6 +985,9 @@ PRAGMA_WARNING_DISABLE_VS(4355)
       boost::asio::ip::tcp::endpoint endpoint = *resolver.resolve(query);
       acceptor_.open(endpoint.protocol());
       acceptor_.set_option(boost::asio::ip::tcp::acceptor::linger(true, 0));
```

Comment:
There is no good reason to change the system's default linger setting.

Reply:
Yes, was an attempt to avoid closed sockets lingering in TIME_WAIT state for too long.
Removed it.

Reply:
Off topic now, but it might make sense to set linger to 0 when dropping a bad peer.